sal-service/src/main/java/org/ow2/proactive/sal/service/service/RepositoryService.java

```diff
@@ -95,6 +95,9 @@ public class RepositoryService {
     @Autowired
     private TaskRepository taskRepository;
 
+    @Autowired
+    private VaultService vaultService;
```

Comment:
From an architectural point of view. The repository service should call only repositories.

Reply:
done